driver/ppm_events_public.h

```diff
@@ -1598,22 +1598,40 @@ enum extra_event_prog_code
 	PPM_SC_X(EPOLL_WAIT_OLD, 379) \
 	PPM_SC_X(RSEQ, 380) \
 	PPM_SC_X(CREATE_MODULE, 381) \
-	PPM_SC_X(PIDFD_GETFD, 382) \
```

Comment:
Not a big deal; i think we can proceed with any solution; i will make sure to provide this fix in its own commit when i open the `merge ppm_tp in ppm_sc` pr so that we can have a discussion there.

This fix **won't** be part of #936 nor #937 .